packages/server/src/sdk/app/rows/tests/internal.spec.ts

```diff
@@ -145,6 +145,7 @@ describe("sdk >> rows >> internal", () => {
                 lastID: 1,
               },
             },
+            _rev: expect.stringMatching("2-.*"),
```

Comment:
Because we're saving the table and updating the `_rev` on it in our new `getNextAutoId` function, this needed adding to this test.